docs/sdks/server-sdks/node.md

```diff
@@ -39,12 +39,12 @@ Initialize the SDK with a SDK key, which can be generated in the Eppo interface.
 import { init } from "@eppo/node-server-sdk";
 
 await init({
-  apiKey: "<API_KEY>",
+  SDKKey: "<SDK_KEY>",
```

Comment:
At the moment, with how the SDKs are, `apiKey` is name specific and cannot be changed to `SDKKey`. This is more in depth work that will have to be updated later. I'll let @leoromanovsky or someone else on engineering confirm. Specifically, the Node SDK requires the key called `apiKey` that takes in a string value that's the SDK key. See this blob from the Node SDK repo: https://github.com/Eppo-exp/node-server-sdk/blob/eb87f5025cd61e47b13b9f4dbda8a7dee9b8920f/src/index.ts#L21

I think it makes sense to update the expected value to it looks like this: `apiKey: "<SDK_key>"` so users know that the expected value is an SDK key and not an API key

Reply:
Sorry, that’s a typo from an overworked Replace function.